packages/connectivity/connectivity_macos/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: connectivity_macos
 description: macOS implementation of the connectivity plugin.
-version: 0.0.2+1
+version: 0.0.3
```

Comment:
pub.dev complains about a package having a version too low. Please make this `0.1.0` at least, so it doesn't impact the score of the plugin in pub.